tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml

```diff
@@ -56,13 +56,13 @@ resources:
           reg_cof: 0.1
       FD:
         master:
-          max: 160
-          mean: 103.4
-          reg_cof: 0.05
+          max: 208
+          mean: 134.42
```

Comment:
The real mean values calculated for the master node in the setup phase were `133.10` and `129.99`. I think `134.42` is a little close to those values and we could get failed runs because of this.

Considering the other values have a higher margin, I would suggest increasing this value to `~170`.